### PR TITLE
RE-146 Rename liberty and mitaka branches

### PR DESCRIPTION
**Pull request by hughsaunders**
liberty-12.2 > liberty
mitaka-13.1 > mitaka

### CODE REVIEW

File: rpc_jobs/rpc_aio.yml [hunk 1]
```diff
@@ -8,12 +8,12 @@
           branches: "kilo.*"
           DEPLOY_TELEGRAF: "yes"
       - liberty:
-          branch: liberty-12.2
+          branch: liberty
           branches: "liberty-.*"
       - mitaka:
           branch: mitaka-13.1
```
Comment (odyssey4me) [inline rpc_jobs/rpc_aio.yml]: Did you miss a rename here?

Comment (hughsaunders) [inline rpc_jobs/rpc_aio.yml]: Yep, forgot to remove the dash, done.

**Comment by hughsaunders** [comment]
Good point @odyssey4me, I deliberately left the selenium branch overrides as they refer to an external repo, but the default is a problem. We can ask @mkam later on if its ok to rename the branches in [rcbops-qe/kibana-selenium](https://github.com/rcbops-qe/kibana-selenium) and [rcbops-qe/horizon-selenium](https://github.com/rcbops-qe/horizon-selenium)

**Comment by mattt416** [comment]
@hughsaunders I've updated existing liberty/newton rcbops-qe and pushed a new branch for mitaka.  @mkam I'll hold off on deleting any existing branches (liberty-12.2, mitaka-13.1, newton-14.x) as I'm not sure if anyone elsewhere in the org are using those branches.

**Pull request merged**